js/adapt-articleView.js

```diff
@@ -27,10 +27,14 @@ define([
         },
 
         preRender: function() {
-
             AdaptArticleView.prototype.preRender.call(this);
-            if (this.model.isBlockSliderEnabled()) this._blockSliderPreRender();
 
+            if (this.model.isBlockSliderEnabled()) {
+                this._blockSliderPreRender();
+                this.$el.addClass('abs-enabled');
```

Comment:
You could define classes in a `className` function:
```js
className: function() {
    return AdaptArticleView.prototype.className.call(this) +=
        this.model.isBlockSliderEnabled() ? ' abs-enabled' : ' abs-disabled';
}
```
Also, is this at odds with line 86?

Reply:
I wouldn't say 'at odds' but there's certainly some duplication when ABS enabled